package.json

```diff
@@ -65,7 +65,7 @@
     "audit:public": "npm audit --registry https://registry.npmjs.org/"
   },
   "dependencies": {
-    "@zowe/imperative": "5.0.0-next.202009221839",
+    "@zowe/imperative": "file:..\\imperative",
```

Comment:
Should the "@zowe/imperative" version be "zowe/imperative#next", so the CI build can find it on GitHub?